src/SMAIAXBackend.Application/DTOs/PolicyCreateDto.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations;
+
+using SMAIAXBackend.Domain.Model.Enums;
+
+namespace SMAIAXBackend.Application.DTOs;
+
+public class PolicyCreateDto(
+    MeasurementResolution measurementResolution,
+    LocationResolution locationResolution,
+    decimal price,
+    Guid smartMeterId)
+{
+    [Required]
+    public MeasurementResolution MeasurementResolution { get; set; } = measurementResolution;
+    [Required]
+    public LocationResolution LocationResolution { get; set; } = locationResolution;
+    [Required]
+    public decimal Price { get; set; } = price;
```

Comment:
Maybe out of scope but not sure how to properly handle price localization. Maybe using PriceInEuro would be better here

Reply:
The [Money Pattern](https://martinfowler.com/eaaCatalog/money.html) would be a perfect fit for this. I created a task on the project board.